src/defs.h

```diff
@@ -154,6 +154,7 @@ typedef struct {
 typedef struct {
     char alias[MAX_VAR_LEN];
     char value[MAX_VAR_LEN];
+    int numeric;
```

Comment:
I just wonder why `numeric` is introduced. Can you eliminate it?

Reply:
Before this patch, `shecc` always views defined symbol as T_numeric. There's chance be T_string such as `#define a "text"`.

Reply:
I see. However, we can still distinguish from string literals on demand.

Reply:
Yes, we can. If we don't want `numeric` as flag, we need to implement another function to distinguish `char*` from literal or numeric. Is that OK?

Reply:
> Yes, we can. If we don't want `numeric` as flag, we need to implement another function to distinguish `char*` from literal or numeric. Is that OK?

It sounds better.